Gordon360/Services/RecIM/SeriesService.cs

```diff
@@ -168,6 +169,11 @@ public async Task<SeriesScheduleViewModel> PutSeriesScheduleAsync(SeriesSchedule
                 }
                 return existingSchedule;
             }
+            // sanitize start/end time dates
+            var defaultDate = new DateTime(1998,12,23);
+            var start = defaultDate.Add(seriesSchedule.DailyStartTime.TimeOfDay);
+            var end = defaultDate.Add(seriesSchedule.DailyEndTime.TimeOfDay);
```

Comment:
Are you using `1998-12-23` as a sentinel value here?

If the `date` portion of `start` and `end` are irrelevant, why not use a `TimeOnly`?

Reply:
Looking back at this, I think I can simplify this along with some of the auto-scheduling logic with more DateTime built in features.

Reply:
I got confused, original comment was referring to a different line of code. 
1.  I'm not using `TimeOnly` is because of the logic of checking whether or not the scheduled time falls within the schedule uses `DateTime`.
2. SQL table is also using DateTime
3. 1998-12-23 is a Date that isnt 0001-01-01 which will thrown an error in SQL since SQL's min DateTime is 1753-01-01. 

Potentially, I could switch implementations to use Time/TimeOnly, but that would only be a minor optimization in terms of space? If that is something that you guys desire, I can make that change.

Reply:
1. You don't actually use the `Date` portion of this `DateTime` when doing that comparison though I believe, you use `Series.StartDate` for the date portion and `Schedule.StartTime` for the time portion, so it would make more sense for those to be a `DateOnly` and `TimeOnly` respectively, right?
2. That is something you can change, no?
3. So are you checking for `1998-12-23` anywhere, or is that date information just completely unused?
4. The point is not optimizing for space, it's representing data accurately. There's a big difference between a `DateTime` and a `TimeOnly` conceptually, and using the former when you actually mean that latter is just asking for weird bugs.

Reply:
Actually, I forgot I switched the `put` route to check only for TimeOfDay. I suppose this is redundant.